packages/wonder-blocks-typography/index.js

```diff
@@ -1,5 +1,40 @@
 // @flow
```

Comment:
So we just ignore index.js files, right?

Reply:
`styleguidist` ignores the `index.js` files.  They're still the entry points though for the modules.